CondFormats/GeometryObjects/interface/HGCalParameters.h

```diff
@@ -0,0 +1,61 @@
+#ifndef CondFormats_GeometryObjects_HGCalParameters_h
+#define CondFormats_GeometryObjects_HGCalParameters_h
+
+#include "CondFormats/Serialization/interface/Serializable.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include<string>
+#include<vector>
+#include<iostream>
+#include<CLHEP/Geometry/Transform3D.h>
+
+class HGCalParameters {
+
+public:
+  
+ HGCalParameters(const std::string& nam): name_(nam) { }
+  ~HGCalParameters( void ) { }
+
+  struct hgtrap {
+    int           lay;
+    float         bl, tl, h, dz, alpha, cellSize;
+  };
+
+  struct hgtrform {
+    int                zp, lay, sec, subsec;
+    bool               used;
+    CLHEP::Hep3Vector  h3v;
+    CLHEP::HepRotation hr;
+  };
```

Comment:
@bsunanda - this object should define what is stored to/retrieved from DB. I think, using vectors of doubles is better.